content/docs/setup/kubernetes/quick-start.md

```diff
@@ -282,20 +282,20 @@ Install Istio's core components. Choose one of the four _**mutually exclusive**_
 
 ## Verifying the installation
 
-1.  Ensure the following Kubernetes services are deployed: `istio-pilot`, `istio-ingress`,
+1.  Ensure the following Kubernetes services are deployed: `istio-pilot`, `istio-ingressgateway`,
 `istio-policy`, `istio-telemetry`, `prometheus`.
 
     ```command
     $ kubectl get svc -n istio-system
-    NAME                       TYPE           CLUSTER-IP   EXTERNAL-IP     PORT(S)                                                               AGE
-    istio-citadel              ClusterIP      30.0.0.119   <none>          8060/TCP,9093/TCP                                                     7h
-    istio-egressgateway        ClusterIP      30.0.0.11    <none>          80/TCP,443/TCP                                                        7h
-    istio-ingressgateway       LoadBalancer   30.0.0.39    9.111.255.245   80:31380/TCP,443:31390/TCP,31400:31400/TCP                            7h
-    istio-pilot                ClusterIP      30.0.0.136   <none>          15003/TCP,15005/TCP,15007/TCP,15010/TCP,15011/TCP,8080/TCP,9093/TCP   7h
-    istio-policy               ClusterIP      30.0.0.242   <none>          9091/TCP,15004/TCP,9093/TCP                                           7h
-    istio-statsd-prom-bridge   ClusterIP      30.0.0.111   <none>          9102/TCP,9125/UDP                                                     7h
-    istio-telemetry            ClusterIP      30.0.0.246   <none>          9091/TCP,15004/TCP,9093/TCP,42422/TCP                                 7h
-    prometheus                 ClusterIP      30.0.0.253   <none>          9090/TCP                                                              7h
+    NAME                       TYPE           CLUSTER-IP       EXTERNAL-IP        PORT(S)                                                               AGE
+    istio-citadel              ClusterIP      100.66.252.72    <none>             8060/TCP,9093/TCP                                                     2d
+    istio-egressgateway        ClusterIP      100.67.131.45    <none>             80/TCP,443/TCP                                                        2d
+    istio-ingressgateway       LoadBalancer   100.67.88.207    ac75b5b826267...   80:31380/TCP,443:31390/TCP,31400:31400/TCP                            2d
```

Comment:
Why did your `EXTERNAL-IP` is `ac75b5b826267..`?

Also I think you only need to update line 285, but do not need to touch other content.

Reply:
We are using LoadBalancer on AWS that why you see an FQDN instead of IP